### ISSUE TITLE
ReactiveUI integration 

### ISSUE THREAD

**Issue by mfkl** [mfkl/LibVLCSharp-readonly#29]

**Comment by jeremyVignelles** [comment]
What do you mean? Integration with reactiveUI by default? a separate package for that?

**Comment by mfkl** [comment]
> Integration with reactiveUI by default? 

No, we don't do integration by default. That'd be making an opinionated choice that's not ours to make for very little benefit to the end user.

> a separate package for that?

Yeah. Somewhat related to https://github.com/mfkl/LibVLCSharp/issues/21, *maybe* some kind of integration with a crossplatform view framework could be interested? ReactiveUI because of the reactive part. There is not much integration to do to use a custom view with a regular MVVM framework like MvvmCross.
